nav2_system_tests/src/behavior_tree/server_handler.cpp

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) 2020 Vinny Ruia
```

Comment:
Is this only for tests? If so, we should move to the tests directory. If not, can you point to an example we'd use this outside of tests?

Reply:
I'm not exactly sure what you mean, can you elaborate?

Reply:
Oh sorry, I thought this was in `nav2_behavior_tree`, not system tests. Carry on :laughing: